src/platform-includes/enriching-events/scopes/scope-callback-param/dotnet.mdx

```diff
@@ -4,8 +4,8 @@ using Sentry;
 // will be tagged with my-tag="my value"
 SentrySdk.CaptureException(new Exception("my error"), configureScope =>
 {
-    scope.SetTag("my-tag", "my value");
-    scope.Level = SentryLevel.Warning;
+    configureScope.SetTag("my-tag", "my value");
```

Comment:
Good catch, thanks 🙏 , but I think we should instead rename `configureScope` to `scope` to be consistent with the other code samples.

Reply:
Yes. Update with scope.